spec/requests/v1/fake_providers_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe V1::FakeProvidersController, type: :request do
   describe 'GET /v1/fake_providers/:username' do
     let(:username) { Faker::Internet.username }
+    let(:contact_name) { username.snakecase.titlecase }
```

Comment:
`.snakecase.titlecase` will remove any hyphens in any names. For example "Smith-Dorian". Probably not a problem, but worth checking that's not an issue.

Reply:
This is the fake API so it doesn't really matter.
We just wanted to automatically return "John Doe" when the username is "john.doe"